### PR TITLE
[BOYSCOUT] Align layout of dbt/git docs by all having a prerequisites…

### PR DESCRIPTION
**Pull request by cfernhout**
… info banner. Fix broken link to merge trigger production job.

### CODE REVIEW

File: docs/deployment_testing/source_control/GitLab.md [hunk 1]
```diff
@@ -5,11 +5,14 @@ description: ""
 pagination_prev: deployment_testing/source_control
 pagination_next: deployment_testing/dbt
 ---
+:::info Prerequisites
```
Comment (williebsweet) [inline docs/deployment_testing/source_control/GitLab.md:8]: It isn't a prereq but needed in the steps that follow

Comment (cfernhout) [reply docs/deployment_testing/source_control/GitLab.md]: @williebsweet So I started off with looking at GitHub. In my mind those are similar to what I wrote at GitLab.
image

File: docs/deployment_testing/dbt/dbt_core.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ description: ""
 pagination_prev: deployment_testing/source_control
 pagination_next: deployment_testing/data_apps
 ---
+
+:::info Prerequisites
+* Using dbt Core requires use of the Datafold SDK. Continue with setup by [generating a Datafold API key](/reference/cloud#create-an-api-key).
+:::
```
[the remark this answers is not in the document]
Comment (cfernhout) [reply docs/deployment_testing/dbt/dbt_core.md:11]: How is it not a prereq if it is required to use dbt Core? Also if felt really out of place as a note under Configuration..

File: docs/deployment_testing/source_control/GitLab.md (hunk 1 above)
Comment (williebsweet) [reply docs/deployment_testing/source_control/GitLab.md]: RE: GitHub - Those are required prior to doing anything underneath.

RE: GitLab - those are required as part of the setup that we are walking them through

Comment (cfernhout) [reply docs/deployment_testing/source_control/GitLab.md]: But don't we have similar prerequisites for GitLab? Like `Datafold Admin role` and `Your GitLab account must be a member of ...`?

**Comment by williebsweet** [comment]
I fixed the core issue that this PR addresses in this new PR (https://github.com/datafold/datafold-docs/pull/157).

**Comment by williebsweet** [comment]
Converted to DRAFT to avoid this from accidentally being merged.

**Comment by williebsweet** [comment]
@cfernhout You raised some good points. Let me spin up some PRs that address your underlying concerns as well as some other things that I noticed.

**Comment by williebsweet** [comment]
Okay - the issues here have been addressed in the three PRs linked above. Closing this one out.

**Pull request closed without merging**